docs/docs/authentication/users.mdx

```diff
@@ -34,7 +34,7 @@ The default role is used when no role is specified in the request. By default, u
 
 You can change what the default role for new users should be at **Settings -> Roles and Permissions**.
 
-### Allowed Roles
+### Default Allowed Roles
 
 Allowed roles are roles the user is allowed to use when making a request. Usually, you would change the role from `user` (the default role) to some other role because you want to use a different role to resolve permissions for a particular request.
```

Comment:
See my other comment. We need to change the text here too.

Reply:
@

> See my other comment. We need to change the text here too.

@szilarddoro my apologies for misunderstanding, now i have changed the text in description part as well.